librz/include/rz_core.h

```diff
@@ -243,9 +243,18 @@ typedef struct rz_core_seek_history_t {
 } RzCoreSeekHistory;
 
 struct rz_core_t {
+	RzAsm *rasm;
+	ut8 ptr_alignment_I;
+	RzAnalysis *analysis;
+	ut8 ptr_alignment_II;
 	RzBin *bin;
+	ut8 ptr_alignment_III;
+	// NOTE: Do not change the order of fields above!
+	// They are used in pointer passing hacks in rz_types.h.
```

Comment:
i'm afraid this will fail in other archs due the endianness

Reply:
The Travis CI passed for PPC though. Let m push again. But yes. In general this is a guess. because I could not find a general way of get the alignment of fields. Cross-platform.

Reply:
hopefully this will be removed with rzarch